index.html

```diff
@@ -14,8 +14,8 @@
     <link rel="stylesheet" type="text/css" href="https://cdn.rawgit.com/Leaflet/Leaflet.markercluster/v0.4.0/dist/MarkerCluster.css">
     <script src="https://cdn.rawgit.com/Leaflet/Leaflet.markercluster/v0.4.0/dist/leaflet.markercluster.js"></script>
     <link href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
-    <link rel="stylesheet" href="http://rawgithub.com/domoritz/leaflet-locatecontrol/gh-pages/dist/L.Control.Locate.min.css" />
-    <script src="http://rawgithub.com/domoritz/leaflet-locatecontrol/gh-pages/dist/L.Control.Locate.min.js"></script>
+    <link rel="stylesheet" href="src/L.Control.Locate.scss" />
```

Comment:
you don't want to point your app at a raw SASS `.scss` file.  this is source code that the project author runs through a [`grunt task`](https://github.com/domoritz/leaflet-locatecontrol/blob/gh-pages/Gruntfile.js#L59) to create normal `.css`.

i'm pretty sure you can just swap in [`L.Control.Locate.css`](https://github.com/domoritz/leaflet-locatecontrol/tree/gh-pages/dist) from the dist folder instead.